docs/contracts.rst

```diff
@@ -435,6 +435,12 @@ Ether (without data).  In such a context, there is very little gas available to
 the function call (to be precise, 2300 gas), so it is important to make fallback functions as cheap as
 possible.
 
+.. warning::
+    The default fallback function will throw an exception (this was different
```

Comment:
I think it is better saying that if "no fallback function is defined". Or explain what "default fallback" means.